querystringsafe_base64.py

```diff
@@ -37,7 +39,9 @@ def encode(to_encode):
         string - like base64, except characters ['+', '/', '='] are
         replaced with ['-', '_', '.'] consequently
     """
-    return urlsafe_b64encode(bytes(to_encode)).replace('=', '.')
+    if PY2:
+        return urlsafe_b64encode(to_encode).replace('=', '.')
+    return urlsafe_b64encode(to_encode).replace(b'=', b'.').decode()
```

Comment:
`'=' == b'='` on Python 2, only `decode` is conditional (if we really want Unicode base64).